download/releaseNotes/releaseNotes7.0.adoc

```diff
@@ -195,3 +195,82 @@ This example also demonstrates real-time producing and consuming of tasks.
 * The documentation is now in AsciiDoc, which improves output quality and makes it easier to contribute.
 * `HardMediumSoftBigDecimalScore`: 3 score levels of BigDecimal scores. Contributed by Brad Hards.
 * The public API is now a solid foundation for the 7.x years.
+
+== Workbench
+
+=== Enhanced solver editor
+
+OptaPlanner Solver editor screen now supports adding all termination types, including composite termination.
+Phase configuration section allows you to tweak Construction Heuristic settings and select Local Search alogorithm
+to optimize your planning problem.
+
+image:7.0/solverEditorScreen.png[Solver editor screen]
+
+=== Guided rule editor integration
+
+Remember the pain of defining a constraint match in rules created by Guided rule editor? The process required creating
+a global variable `scoreHolder` and using `Free form line` input to insert Java code to modify the score.
+
+[source,java,options="nowrap"]
+----
+scoreHolder.addSoftConstraintMatch(kcontext, -$computer.cost);
+----
+
+The procedure is a lot simpler now. A correct list of all available score operations is provided in the action list of
+a guided rule, based on the selected score type in your planning solution object. Just define the value of a constraint.
+
+image:7.0/guidedRule.png[Guided rule editor]
+
+=== Difficulty comparator definition
+
+OptaPlanner domain editor now provides a way to specify how difficult is to assign a planning entity.
```

Comment:
keep it concise
"... editor can now specify ..."

grammar: "how difficult is to assign" is not proper english